src/FsAutoComplete/LspHelpers.fs

```diff
@@ -18,12 +18,23 @@ module FcsRange = FSharp.Compiler.Text.Range
 type FcsPos = FSharp.Compiler.Text.Position
 module FcsPos = FSharp.Compiler.Text.Position
 
+// type LspPos = Ionide.LanguageServerProtocol.Types.Position
+// type LspRange = Ionide.LanguageServerProtocol.Types.Range
+
+module Lsp = Ionide.LanguageServerProtocol.Types
+
+module LspRange =
+  // static member Create (start: LspPos) (end_: LspPos) : LspRange =
+  //   { Start = start; End = end_ }
+  let Zero: Lsp.Range =
+    { Start = { Line = 0; Character = 0 }
+      End = { Line = 0; Character = 0 } }
```

Comment:
I think this should be removed. I was trying to figure out which things were 0 and 1 based.